tree/ntuple/src/RField.cxx

```diff
@@ -602,13 +614,19 @@ ROOT::RRecordField::RRecordField(std::string_view fieldName, std::vector<std::un
 {
    fTraits |= kTraitTrivialType;
    fOffsets.reserve(itemFields.size());
+   std::unordered_set<std::string> fieldNames;
```

Comment:
For the local `std::unordered_set`, we probably only need `std::string_view` as the item type and avoid the string copies